examples/restaurant_demo/restaurant.py

```diff
@@ -98,80 +101,92 @@ def get_private_counts_per_hour(self, privacy_budget: float) -> dict:
+                list(day_visits[day_visits["Day"] == day]["Day"]))
+
         return day_counts
 
     def get_non_private_sum_revenue(self) -> dict:
         day_revenue = dict()
 
         for day in self._day_visits["Day"].unique():
-            day_revenue[day] = self._day_visits[self._day_visits["Day"] == day]["Money spent (euros)"].sum()
-        
-        return day_revenue  
+            day_revenue[day] = self._day_visits[self._day_visits["Day"]
+                                                == day]["Money spent (euros)"].sum()
+
+        return day_revenue
 
     def get_private_sum_revenue(self, privacy_budget: float) -> dict:
         # Pre-process the data set: limit the number of days contributed by a visitor to SUM_MAX_CONTRIBUTED_DAYS
-        day_visits = bound_visits_per_week(self._day_visits, SUM_MAX_CONTRIBUTED_DAYS)
+        day_visits = bound_visits_per_week(
+            self._day_visits, SUM_MAX_CONTRIBUTED_DAYS)
 
         day_revenue = dict()
 
         x = dp.BoundedSum(LN_3, MIN_EUROS_SPENT, MAX_EUROS_SPENT_1)
 
         for day in day_visits["Day"].unique():
-            day_revenue[day] = int(x.result(list(day_visits[day_visits["Day"] == day]["Money spent (euros)"])))
-        
-        return day_revenue 
+            day_revenue[day] = int(
+                x.result(list(day_visits[day_visits["Day"] == day]["Money spent (euros)"])))
+
+        return day_revenue
 
     def get_private_sum_revenue_with_preaggregation(self, privacy_budget: float) -> dict:
         # Pre-process the data set: limit the number of days contributed by a visitor to SUM_MAX_CONTRIBUTED_DAYS
-        day_visits = bound_visits_per_week(self._day_visits, SUM_MAX_CONTRIBUTED_DAYS)
+        day_visits = bound_visits_per_week(
+            self._day_visits, SUM_MAX_CONTRIBUTED_DAYS)
 
         day_revenue = dict()
 
         x = dp.BoundedSum(LN_3, MIN_EUROS_SPENT, MAX_EUROS_SPENT_2)
 
         for day in day_visits["Day"].unique():
             # For each visitor, pre-aggregate their spending for the day.
-            visits_on_day = day_visits[day_visits["Day"] == day][["VisitorId", "Money spent (euros)"]]
+            visits_on_day = day_visits[day_visits["Day"] == day][[
+                "VisitorId", "Money spent (euros)"]]
             visitor_to_spending = dict()
 
             for visitor in visits_on_day["VisitorId"].unique():
-                visitor_to_spending[visitor] = visits_on_day[visits_on_day["VisitorId"] == visitor]["Money spent (euros)"].sum()
+                visitor_to_spending[visitor] = visits_on_day[visits_on_day["VisitorId"]
+                                                             == visitor]["Money spent (euros)"].sum()
 
             spending = list(visitor_to_spending.values())
-            
+
             day_revenue[day] = int(x.result(spending))
 
         return day_revenue
 
+
 def bound_visits_per_week(df, limit):
     """ Make sure that each visitor only contributes at most 3 visits per week,
```

Comment:
Here, you're using triple quotes/a delimiter (""") instead of a hash (#) for the docstring. 
For consistency, I think we should have all docstings with """.
(This can potentially help with auto-generating (API) docs.)

Reply:
Gotcha, sounds good!